include/rogue/interfaces/stream/FrameIterator.h

```diff
@@ -106,6 +108,15 @@ namespace rogue {
                 */
                uint32_t remBuffer();
 
+               //! Get payload size
```

Comment:
I am not a fan of this and only adds a small amount of information to the below error.